scripts/cnn_variant_wdl/jsons/cnn_score_variants_travis.json

```diff
@@ -0,0 +1,22 @@
+{
+  "CNNScoreVariantsWorkflow.bam_file": "/home/travis/build/broadinstitute/gatk/src/test/resources/large/VQSR/g94982_chr20_1m_10m_bamout.bam",
+  "CNNScoreVariantsWorkflow.bam_file_index": "/home/travis/build/broadinstitute/gatk/src/test/resources/large/VQSR/g94982_chr20_1m_10m_bamout.bai",
+  "CNNScoreVariantsWorkflow.calling_intervals": "/home/travis/build/broadinstitute/gatk/src/test/resources/large/VQSR/contig20_1m_10m.interval_list",
+  "CNNScoreVariantsWorkflow.gatk_docker": "__GATK_DOCKER__",
```

Comment:
This is a placeholder?

Reply:
yes because the actual docker will depend on the commit hash that is being pushed to travis.